website/docs/guidelines/pull-request/format.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: Format
+sidebar_position: 1
+---
+
+# Format for Pull Requests
+
+This document outlines the required format for pull requests to ensure clarity and consistency.  
+Adhering to these guidelines will facilitate smoother reviews and better communication among team members.
+
+## Title
+
+The title of a pull request should be concise and meaningful, summarizing the changes made.  
+Keep the title within 72 characters to avoid truncation by GitHub, ensuring that the full title is visible in notifications and lists, which aids in quick identification and understanding of the PR's purpose.  
+Avoid including references to tracking systems (e.g., Jira task IDs) in the title, as they can clutter it and may not be useful for external reviewers.
+
+## Description
+
+The description of a pull request must explain the rationale behind the modifications, offering a brief overview of the updates along with any relevant context.  
+It should include:
+
+- A summary of the problem being solved or the feature being added
+- If applicable, include a reference to the tracking system at the end of the description using `Resolves #<Jira task ID>` (e.g., `Resolves #DX-1234`)
+
+### Tracking System References
+
+When referencing an activity, such as a Jira task, in a pull request, use one of [GitHub's supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) followed by the task ID.
```

Comment:
I believe these keywords are meaningful to link PRs with GitHub issues, but we can use any word to link a Jira issue instead. Am I wrong?

Reply:
you are right. the key of the Jira task is enough to link the PR to Jira.

The idea of using the GitHub conventions is just to use something is already used by others for a similar use case. we borrow their keywords to have meaningful text within the PR description.

In terms of output, the following description's examples are working in the same way:
```
This is a description and closes #<JiraId>
```

```
<JiraId>
```

```
This is <JiraId>
```

Reply:
@Krusty93 I’ve updated the description to clarify the use of GitHub keywords. Please let me know if it makes sense now or causes any confusion. If needed, I’m happy to update the description again.

Reply:
Looks good to me